Assets/Scripts3D/Weapons/Weapon.cs

```diff
@@ -43,7 +72,39 @@ private void Shoot()
         {
             return;
         }
+    }
+
+    void RotationSway()
```

Comment:
This method name doesn't indicate what it does. Rename something like `ApplyRotationSway()', `UpdateRotationSway()' etc

Reply:
Maybe `ApplyCameraRotationSway()`? There's no downside to being as explicitly clear as possible